projects/hipdnn/backend/tests/descriptors/mocks/MockEnginePluginResourceManager.hpp

```diff
@@ -56,4 +54,3 @@ class MockEnginePluginResourceManager : public EnginePluginResourceManager
 };
 
 }
```

Comment:
```suggestion
} // namespace hipdnn_backend::plugin
```
Nit, but for consistency.